posts/2024-02-15-Windows-7.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "New Windows 7 targets and tier 1 deprecation"
+author: Chris Denton
```

Comment:
This should probably have a "on behalf of X team"

Reply:
Fair enough. I couldn't find any guidance so just clicked a few other posts on this repo. I guess it would be "The Compiler Team" as that's who ultimately approved the MCP?

Reply:
We haven't always been consistent about following this but yeah, "The Compiler Team" is fine.